base/src/main/java/com/smartdevicelink/proxy/rpc/RadioControlData.java

```diff
@@ -290,4 +294,23 @@ public void setSisData(SisData sisData) {
     public SisData getSisData() {
         return (SisData) getObject(SisData.class, KEY_SIS_DATA);
     }
+
+    /**
+     * Sets the availableHdChannels portion of the RadioControlData class
+     *
+     * @param availableHdChannels List of available hd sub-channel indexes, empty list means no Hd channel is available, read-only.
+     */
+    public void setAvailableHdChannels(List<Integer> availableHdChannels){
```

Comment:
Thanks for the review! Other read only parameters in the project have both get and set methods, like seen in the example below.  Would it be preferred for us to explicitly mention in the java docs that the parameter is read only similar to the example below?

https://github.com/smartdevicelink/sdl_java_suite/blob/80d6737382c43dff10358198b5a4cccf3b802cfc/base/src/main/java/com/smartdevicelink/proxy/rpc/RadioControlData.java#L122-L139

Reply:
I think explicitly mentioning that will be alright in the java docs per the sample above. We should also make note when we write the actual documentation for it too. I remember testing before and the read only parameters were confusing with no docs. @bilal-alsharifi if there is ever a RC manager or something of that nature it might need a public.

Reply:
Agree with the JavaDoc method. Since the RPC message itself can have the property written to it, the setter is fine. It is only in the RPC spec that mentions this value can't be modified on the module itself.

Reply:
Okay, that sounds good to me too. I was not aware of the setters for the other read-only params. Thanks, @mrapitis, @BrettyWhite, and @joeygrover.